### ISSUE TITLE
Sort changes by Priority

### ISSUE THREAD

**Issue by buzea** [uwolfer/gerrit-intellij-plugin#311]
Hello,

Could you please take into consideration the feature of sorting the changes by the columns available in the Intellij-Gerrit View? 
I would really like to sort my changes by Priority

**Comment by uwolfer** [comment]
"Priority" is a custom column in your Gerrit setup, right? It is no available e.g. here: https://gerrit-review.googlesource.com/
Anyway, you cannot order by column in the web UI, right?I think it is not supported by the Gerrit REST API: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html

So ordering would only be possible client side with the currently loaded changes - if there are more than 25 changes displayed, it would not make sense.

If you see a better way to do this job, please tell me.

**Comment by buzea** [comment]
Yes, you are right on all aspects. It would be however be a very helpful feature for my team. We actually use this Priority column to manage or work and changes with higher priority go first in review. 
So it is possible to make just the loaded changes sortable based on a specific column on the client side?
